### ISSUE TITLE
Document which GenericMap methods work with dask

### ISSUE THREAD

**Issue by dstansby** [sunpy/sunpy#7224]
### Describe the feature

https://github.com/sunpy/sunpy/pull/7100 added some nice tests to work out which GenericMap methods work nicely with dask arrays. It would be good to add a bit of user facing documentation that lists which methods work/which ones don't with dask arrays.

**Comment by wtbarnes** [comment]
Good idea. I almost see this in the same category as a "known issue", i.e. provide a list of methods we know *don't* preserve laziness and explain why that is currently.

**Comment by Rjchauhan18** [comment]
Hello @dstansby sir and @wtbarnes  sir , I want to participate in Gsoc 2024 in sunpy repositery. so ,Can i work on this issue ?

>Good idea. I almost see this in the same category as a "known issue", i.e. provide a list of methods we know don't preserve laziness and explain why that is currently.
Can you please tell me how to check laziness in test mesthods ?

**Comment by nabobalis** [comment]
> Hello @dstansby and @wtbarnes ,Can i work on this issue ?

You are free to work on this issue. 

> > Good idea. I almost see this in the same category as a "known issue", i.e. provide a list of methods we know don't preserve laziness and explain why that is currently.
> 
> Can you please tell me how to check laziness in test mesthods ?

If I recall, the return from the method will not be computed value but some of dask delayed type.
For example if you see https://github.com/sunpy/sunpy/blob/main/sunpy/map/tests/test_mapbase_dask.py#L50 for reference.

**Comment by tanishy7777** [comment]
@nabobalis Hey, can I work on this?

**Comment by tanishy7777** [comment]
> Go ahead.
The documentation should be added in the `sunpy/map/tests/test_mapbase_dask.py` file right?

**Comment by nabobalis** [comment]
> > Go ahead.
> 
> The documentation should be added in the `sunpy/map/tests/test_mapbase_dask.py` file right?
> 
> 

No, as no one will look in there. It should be added to the methods documentation string and possibly a separate table in the map API page
